cmd/panfs.go

```diff
@@ -690,6 +734,17 @@ func (fs *PANFSObjects) DeleteBucket(ctx context.Context, bucket string, opts De
 	// Delete all bucket metadata.
 	deleteBucketMetadata(ctx, fs, bucket)
 
+	if fs.panclient != nil {
```

Comment:
It seems that before process delete request we should ask configuration agent to check whether bucket with provided name exists